back/db/migrate/20231130093345_add_phase_foreign_keys.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class AddPhaseForeignKeys < ActiveRecord::Migration[6.1]
+  def change
+    %i[baskets polls_responses polls_questions volunteering_causes surveys_responses].each do |table|
+      add_foreign_key table, :phases, column: :phase_id, on_delete: :nullify
```

Comment:
Used nullify here as there was no previous foreign key when it was participation_context, so this keeps the same behaviour.